.vscode/settings.json

```diff
@@ -1,3 +1,8 @@
 {
-    "python.linting.pylintEnabled": false
+    "python.linting.pylintEnabled": false,
+    "editor.fontSize": 16,
+    "files.exclude": {
```

Comment:
Oh, this took a font size change I didn't want it to. What do you think about adding it to gitignore instead? Or maybe get it in a good 'default' state and then gitignoring it so further changes don't get passed up?

Reply:
Yeah, let's gitignore it for now

Reply:
Well that took a few more commits than I had planned but 248ea25 ought to do it. Since git 'knew' about .vscode/settings.json just gitignoring it wasn't enough, so I followed the instructions [from here](https://stackoverflow.com/questions/11451535/gitignore-is-ignored-by-git)
```
git rm -r --cached .
git add .
git commit -m "fixed untracked files"
```

It worked - but it also removed these, so I wonder if we had a few other orphaned files kicking around. Nothing seems to have broken after they were removed, at least.
```
Changes to be committed:
  (use "git reset HEAD <file>..." to unstage)

	deleted:    .vscode/settings.json
	deleted:    common/components/common/tags/TagCategory.jsx
	deleted:    common/components/common/tags/TagSelectWrapper.jsx
	deleted:    common/components/common/tags/TagSelector.jsx
	deleted:    common/components/common/tags/TagsDisplay.jsx
```